src/com/kainos/buildasub/oop/Sub.java

```diff
@@ -1,17 +1,19 @@
 package com.kainos.buildasub.oop;
 
+import java.util.ArrayList;
+
 /**
  * Created by trainee on 17/10/16.
  */
 public class Sub {
     private String meat;
     private String bread;
-    private String[] salad;
+    private ArrayList <String> salad;
```

Comment:
you can do this, or you can convert the ArrayList in the main method to an array when you instantiate the Sub